dependencies.list

```diff
@@ -1,4 +1,4 @@
 VERSION=3.0.0-beta.3
 REALM_CORE_VERSION=3.0.0-rc5
 REALM_SYNC_VERSION=2.0.0-rc15
-REALM_OBJECT_SERVER_VERSION=2.0.0-rc2
+REALM_OBJECT_SERVER_VERSION=2.0.0-alpha.25
```

Comment:
Doesn't ROS v2.0.0-alpha.25 require that we use sync v2.0.0-rc18?

Reply:
Perhaps. I can't seem to find the correct branch in the ROS repo. I'll check again when I get home.

Reply:
rc18 doesn't seem to be published; the build scripts can't find it.

Reply:
I think they may have fixed the file extension of the sync archive back to being .xz like it should have been. Try tweaking the extension here: https://github.com/realm/realm-cocoa/blob/master-3.0/build.sh#L329